src/components/Dashboard.tsx

```diff
@@ -94,38 +85,89 @@ const TemplateComponent: React.FC<{ template: TemplateInList }> = ({ template })
   )
 }
 
-const FilterComponent: React.FC<{
+const PanelComponent: React.FC<{
   template: TemplateDetails
-  filter: Filter
-  setFiltersCount: Function
-}> = ({ template, filter, setFiltersCount: setLoadingFilters }) => {
+  filters: Filter[]
+}> = ({ template, filters }) => {
   const templateType = template.code
-  const { loading, applicationCount } = useListApplications({
-    type: templateType,
-    perPage: 1,
-    ...filter.query,
-  })
+  const [loadedFiltersCount, setLoadedFiltersCount] = useState(0)
+  const [totalMatchFilter, setTotalMatchFilter] = useState<{ [key: number]: number }>(
+    filters.reduce((totalPerFilter, filter) => ({ ...totalPerFilter, [filter.id]: 0 }), {})
+  )
+
+  const arrayFilters = filters.reduce((arrayFilters: { [key: string]: string }[], element) => {
+    if (typeof element.query === 'object') {
+      const queryObj = element.query as { [key: string]: string }
+      arrayFilters.push(queryObj)
+    }
+    return arrayFilters
+  }, [])
+
+  const queryMultiFilters = {
```

Comment:
Similarly, this name doesn't really capture what it is. We've already been using `graphQLFilterObject` for similar objects elsewhere, so I'd suggest using something similar, (cos that's actually what it is).